group-generators/helpers/data-providers/index.ts

```diff
@@ -19,6 +19,7 @@ import HiveInterfaceSchema from "./hive/interface-schema.json";
 import { JsonRpcProvider } from "./json-rpc";
 import { LensProvider } from "./lens";
 import lensInterfaceSchema from "./lens/interface-schema.json";
+import { OtterspaceProvider } from "./otterspace";
```

Comment:
You need to import your `interfaceSchema` here, in order to put it in `dataProvidersInterfacesSchemas` array